controllers/user.js

```diff
@@ -354,7 +354,7 @@ async function getBreachStats(req, res) {
     });
   }
   const fxaResponse = await FXA.verifyOAuthToken(req.token, FXA_MONITOR_SCOPE);
-  if (fxaResponse.name === "HTTPError") {
+  if (fxaResponse.name === "HTTPError" || !fxaResponse.body.scope.includes(FXA_MONITOR_SCOPE)) {
```

Comment:
nit: Semi-related, but it doesn't look like `FXA.verifyOAuthToken()` uses the `scope` argument anymore.

https://github.com/mozilla/blurts-server/blob/c9dc4941cddd2d97db6a1eea9317536a27bb5617/lib/fxa.js#L33-L40

Not sure if it's any cleaner to remove that argument to reduce visual clutter (and unnecessary arg passing).

Reply:
I recommend removing it to reduce possible future confusion, esp. since there's no test that would prevent a future refactor from assuming that `verifyOAuthToken` checks the passed-in scope.